doc/musig-spec.mediawiki

```diff
@@ -129,15 +129,22 @@ The algorithm ''NonceAgg(pubnonce<sub>1..u</sub>)'' is defined as:
 ===== Note on ''is_infinite(R'<sub>i</sub>)'' =====
 
 If ''is_infinite(R'<sub>i</sub>)'' there is at least one dishonest signer (except with negligible probability).
-If we would fail here, we will never be able to determine who it is.
-Therefore, we should continue such that the culprit is revealed when collecting and verifying partial signatures.
+If we fail here, we will never be able to determine who it is.
+Therefore, we continue so that the culprit is revealed when collecting and verifying partial signatures.
+
 However, dealing with the point at infinity requires defining a serialization and may require extra code complexity in implementations.
-Instead, we set the aggregate nonce to some arbitrary point, the generator.
+Instead of incurring this complexity, we make two modifications (compared to the MuSig2* appendix in the [https://eprint.iacr.org/2020/1261 MuSig2 paper]) to avoid infinity while still allowing us to detect the dishonest signer:
+* In ''NonceAgg'', if an output ''R'<sub>i</sub>'' would've been infinity, instead output the generator (an arbitrary choice).
+* In ''Sign'', implicitly disallow the input ''aggnonce'' to contain infinity (since the serialization format doesn't support it).
+
+The entire ''NonceAgg'' function (both the original and modified version) only depends on publicly available data (the set of public pre-nonces from every signer).
+In the security proof, we consider ''NonceAgg'' to be performed by an untrusted party; thus modifications to ''NonceAgg'' do not affect the security of the scheme.
```

Comment:
```suggestion
In the unforgeability proof, ''NonceAgg'' is considered to be performed by an untrusted party; thus modifications to ''NonceAgg'' do not affect the unforgeability of the scheme.
```
(unforgeability is more precise -- in particular because "security" in this paragraph could also refer to the ability to spot the dishonest signer, what people call "identifiable aborts")
(Active -> passive because the "we" sounds like the model is something that we make up here. But it's just a common thing in the paper and in the spec here.)